src/components/dev-dot-content/mdx-components/mdx-alert/docs.mdx

```diff
@@ -129,12 +124,23 @@ opening / closing tags.
 ### Multiple child elements
 
 <MdxInlineAlert type="warning">
+
 	Liquorice cake marzipan danish brownie
 	shortbread dessert. Lollipop gingerbread bear claw muffin croissant cake caramels
 	halvah.
 
-    Bonbon pie jelly beans jelly beans oat cake muffin. Jelly
-    beans dragée sugar plum lollipop lemon drops. Cake donut cookie sugar plum
-    shortbread caramels.
+	Bonbon pie jelly beans jelly beans oat cake muffin. Jelly
+	beans dragée sugar plum lollipop lemon drops. Cake donut cookie sugar plum
+	shortbread caramels.
+
+</MdxInlineAlert>
+
+## Deprecated
+
+The authoring style guide discourages use of the `Highlight` component. Do not use.
 
+### Highlight
+
+<MdxInlineAlert type="highlight">
+	To provide general or promotional information to the user prominently.
 </MdxInlineAlert>
```

Comment:
```suggestion
```
We can just remove this entirely from the mdx authoring. Since it was never officially released, I don't think we need to make a deprecated notice.

Reply:
I'll say that it was expressed by education managers that they wanted a colored alert type, such as the highlight. It was set up in original requirements to provide this. Before removing fully, I want to make sure all education managers feel okay with this.

Reply:
I'd still like to remove this deprecation notice from the docs before merging!